docs/bokeh/source/docs/first_steps/first_steps_1.rst

```diff
@@ -60,61 +60,61 @@ right of the plot to explore:
 Follow these steps to recreate this simple line chart:
 
 1. Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open
-it in a code editor of your choice (such as `Sublime Text`_,`Visual Studio Code`_ , etc.).
+   it in a code editor of your choice (such as `Sublime Text`_, `Visual Studio Code`_ , etc.).
 
 .. _Sublime Text: https://www.sublimetext.com/
 .. _Visual Studio Code: https://code.visualstudio.com/
 
 2. As the first line of your new Python script, import the necessary functions from the |bokeh.plotting| module:
 
-    .. code-block:: python
+.. code-block:: python
 
-        from bokeh.plotting import figure, show
+    from bokeh.plotting import figure, show
```

Comment:
@tcmetzger can you comment on this? Did you intend for the code blocks to be indented under each bullet?

Reply:
Yes, the idea is to have those blocks indented to be part of the bullet. This worked nicely in the past (docs for Bokeh 2.4, for example):
![image](https://github.com/bokeh/bokeh/assets/39711796/9db32977-5710-4b8e-95cd-b937e7ae4e09)

In the docs that are currently online, these indented blocks don't render correctly anymore:
![image](https://github.com/bokeh/bokeh/assets/39711796/9e3fd49f-30a6-47d4-809c-4de9d3832133)

Something went wrong here at some point (potentially the PR that added the new first item here, or an update in Sphinx/theme). I agree that we should fix this so that this section again looks like it used to in the older versions (indentation-wise, the text that was added about creating and opening the file should stay, of course)

Reply:
I looked into this some more and I believe the indents are still too large - Sphinx requires three (instead of 4) spaces for indents. This is what this section should look like to render correctly:
```

1. Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open
   it in a code editor of your choice (such as `Sublime Text`_, `Visual Studio Code`_ , etc.).

.. _Sublime Text: https://www.sublimetext.com/
.. _Visual Studio Code: https://code.visualstudio.com/

2. As the first line of your new Python script, import the necessary functions from the |bokeh.plotting| module:

   .. code-block:: python

      from bokeh.plotting import figure, show

3. Define two lists containing the data for your line chart:

   .. code-block:: python

      # prepare some data
      x = [1, 2, 3, 4, 5]
      y = [6, 7, 2, 4, 5]

4. Use the |figure| function to create your plot. Pass the following arguments:

   * ``title``: the title of your line chart (optional)
   * ``x_axis_label``: a text label to put on the chart's x-axis (optional)
   * ``y_axis_label``: a text label to put on the chart's y-axis (optional)

   .. code-block:: python

      # create a new plot with a title and axis labels
      p = figure(title="Simple line example", x_axis_label='x', y_axis_label='y')

5. Add a line graph to the plot you just created, using the
   :func:`~bokeh.plotting.figure.line` function. Pass the following arguments:

   * your lists ``x`` and ``y`` containing the data
   * ``legend_label``: a string to label the line graph with (optional)
   * ``line_width``: define the line width (in pixels, optional)

   .. code-block:: python

      # add a line renderer with legend and line thickness to the plot
      p.line(x, y, legend_label="Temp.", line_width=2)

6. Finally, use the |show| function to generate your graph and
   open a web browser to display the generated HTML file.

   .. code-block:: python

      # show the results
      show(p)

7. From the command line, run the Python script you just created. For example:

   .. code-block:: sh

      python simple_line_chart.py

```

This way, the indentations look correct:
![image](https://github.com/bokeh/bokeh/assets/39711796/926ddb61-6418-4627-bf43-013eb1dd74c1)

Reply:
> Sphinx requires three (instead of 4) 

I'm probably more a Sphinx/ReST fan than the average Joe, but geez that's annoying. I wonder if there is some linter tool that would catch these sorts of things.

Reply:
I was playing on this a bit more and I want to add, that there is no correct number of white spaces. The rule seems to be, that the code block has to be on aligned with the text above.

Here is an example to illustrate it. Both example will generate the same output. 

```rest
1. Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open
   it in a code editor of your choice (such as `Sublime Text`_, `Visual Studio Code`_ , etc.).

.. _Sublime Text: https://www.sublimetext.com/
.. _Visual Studio Code: https://code.visualstudio.com/

2. As the first line of your new Python script, import the necessary functions from the |bokeh.plotting| module:

   .. code-block:: python

       from bokeh.plotting import figure, show
```
 and

```rest

1.     Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open
        it in a code editor of your choice (such as `Sublime Text`_, `Visual Studio Code`_ , etc.).

.. _Sublime Text: https://www.sublimetext.com/
.. _Visual Studio Code: https://code.visualstudio.com/

2.     As the first line of your new Python script, import the necessary functions from the |bokeh.plotting| module:

       .. code-block:: python

           from bokeh.plotting import figure, show
```